pkg/provision/automount/common.go

```diff
@@ -78,6 +79,16 @@ func ProvisionAutoMountResourcesInto(podAdditions *v1alpha1.PodAdditions, api sy
 	return nil
 }
 
+func ProvisionServiceAccountTokensInto(podAdditions *v1alpha1.PodAdditions, workspace *common.DevWorkspaceWithConfig) error {
+	saTokenVolumeMounts, saTokenVolumes, err := getSATokensVolumesAndVolumeMounts(workspace.Config.Workspace.ServiceAccountTokens)
+	if err != nil {
+		return err
+	}
+	podAdditions.VolumeMounts = append(podAdditions.VolumeMounts, saTokenVolumeMounts...)
+	podAdditions.Volumes = append(podAdditions.Volumes, saTokenVolumes...)
+	return nil
+}
+
```

Comment:
I'm not sure this function belongs in the `automount` package. Mounting generally-configured SA tokens is fairly different functionality from what automount currently does, and it does not seem that any of the functions included in supporting this overlap with other automount functions. 

I think this should be moved to a different package (though, of course, naming is hard)

Reply:
Sounds good, will find somewhere else to put this :+1: